src/test/java/org/apache/commons/imaging/formats/tiff/datareaders/DataReaderStripsTest.java

```diff
@@ -24,7 +24,7 @@
     @Test
     public void testApplyPredictor() throws Exception {
         final int[] bitsPerPixel = {1,2,3};
-        final DataReaderStrips strips = new DataReaderStrips(null, null, 3, bitsPerPixel, 2, 4, 0, 3, 1, 1, null, 2, null);
+        final DataReaderStrips strips = new DataReaderStrips(null, null, 3, bitsPerPixel, 2, 4, 0, 3, 1, 1, 1, null, 2, null);
```

Comment:
Can we use the constant `TiffTagConstants.PLANAR_CONFIGURATION_VALUE_CHUNKY` (not sure if the `1` here corresponds to that constant's value?)?

Reply:
Agreed. Good catch.